### PR TITLE
Rearrange badges, add codefactor

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 [![gradle plugin](https://img.shields.io/badge/Gradle-1.0.0--RC16-blue.svg)](https://plugins.gradle.org/plugin/io.gitlab.arturbosch.detekt)
+<!-- Version needs to be hardcoded until rc phase is over. Change to '-'RC for semantic versioning broke the badge. [![gradle plugin](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/io/gitlab/arturbosch/detekt/detekt-gradle-plugin/maven-metadata.xml.svg?label=gradle&style=flat-square)](https://plugins.gradle.org/plugin/io.gitlab.arturbosch.detekt)-->
 
+[![build status](https://travis-ci.org/arturbosch/detekt.svg?branch=master)](https://travis-ci.org/arturbosch/detekt)
+[![build status windows](https://ci.appveyor.com/api/projects/status/3q9g98vveiul7yut/branch/master?svg=true)](https://ci.appveyor.com/project/arturbosch/detekt)
+[![codecov](https://codecov.io/gh/arturbosch/detekt/branch/master/graph/badge.svg)](https://codecov.io/gh/arturbosch/detekt)
```
Comment (schalkms) [inline README.md]: That doesn't work, however.

Comment (arturbosch) [reply README.md]: why, you can click at the rich diff and see that the badges get all rendered?

Comment (schalkms) [reply README.md]: The link works, but the codecov report itself is not working for detekt.

Comment (arturbosch) [reply README.md]: It's the link provided by codecov itself idk why it says unknown ... :(

**Pull request merged**